Queries.sql

```diff
@@ -0,0 +1,83 @@
+SELECT
+  Operators.model_name,
+  tensor_label,
+  tensor_shape,
+  tensor_type
+FROM
+  Operators
+JOIN
+  Tensors
+ON
+  Operators.model_name = Tensors.model_name
+  AND operator_id IN UNNEST(from_operator_ids)
+WHERE
+  operator_type = "Input_Placeholder"
```

Comment:
Some description about each query would be helpful.

Reply:
Fixed in latest commit

Reply:
Added in a previous commit